person.rb

```diff
@@ -1,3 +1,4 @@
+require_relative './nameable'
 class Person
   attr_accessor :name, :age
   attr_reader :id
```

Comment:
- [ ] Great work implementing the requirements for this project. Kindly make the `Person` class inherit from the `Nameable` class based on the project requirements. 👍